src/components/backend-ai-signup.ts

```diff
@@ -308,7 +317,10 @@ export default class BackendAiSignup extends BackendAIPage {
     if (inputFieldsValidity.includes(false)) {
       return;
     }
-    const token = this.tokenInput.value;
+    let token;
+    if (!this.allowSignupWithoutConfirmation) {
+      token = this.tokenInput.value;
+    }
```

Comment:
You don't need to do this because 335 line does something.  Please revert this part

Reply:
I added this code because token input is appered when allowSignupWithoutConfirmation is false like below code.
https://github.com/lablup/backend.ai-webui/blob/0c0e445222506b25f1b8c3dd11a264b8f6bf5e5f/src/components/backend-ai-signup.ts#L531-L543

Before code token value is always get input value.
But I think It is better to change to add token at body value when allowSignupWithoutConfirmation is false.